src/containers/edm/propertytypes/PropertyTypeDetailsContainer.js

```diff
@@ -4,13 +4,14 @@
 
 import React from 'react';
 
-import { Map } from 'immutable';
+import Immutable, { Map, List } from 'immutable';
```

Comment:
You don't need to import the default `Immutable`; you can just add `Set` to the list.

```
import { Map, List, Set } from 'immutable';
```